pnet/Private-Networks-PSK-V1.md

```diff
@@ -0,0 +1,75 @@
+## Pre-shared Key Based Private Networks in IPFS
+
+This document describes the first version of private networks (PN) featured in IPFS.
+
+For the first implementation, only pre-shared key (PSK) functionality is available, as the Public Key Infrastructure approach is much more complex and requires more technical preparation.
+
+It was implemented as an additional encryption layer before any IPFS traffic, and was designed to leak the absolute minimum of information on its own. All traffic leaving the node inside a PN is encrypted and there is no characteristic handshake.
+
+### Interface
+
+An IPFS node or libp2p swarm is either in a a public network, or it is a member of a private network.
```

Comment:
`s/swarm/node`

Describe public as "anyone can join" and private as "only authorized people can be part of"